internal/tsc_wrapped/file_cache.ts

```diff
@@ -103,12 +103,16 @@ export class FileCache<T = {}> {
     this.debug('updating digests:', digests);
     this.lastDigests = digests;
     this.cannotEvict = false;
-    for (const [filePath, newDigest] of digests.entries()) {
+    const fileEntries = Array.from(digests.keys());
+    for (let i = 0; i < fileEntries.length; i++) {
```

Comment:
have we measured that `for of` is actually still slow in v8? 

v8 team has been focused on perf improvements of common js patterns:
- https://v8.dev/blog/fast-for-in
- https://v8.dev/blog/spread-elements

have we measured this using the perf harness that @filipesilva's built?

Reply:
Interesting, looks like im pretty out of date on this one.
Looking at this here: https://jsperf.com/for-of-vs-for-loop - specifically the section at the bottom showing this test on Chrome versions going back to 49 when `for` was back then significantly faster, you can see the improving trend over the past few versions which would aling with those articles.

Right now on Chrome 70 on my system, it appears that `for of` is actually ~20% faster.
Which I wouldn't take as a hard fact that `for of` is 20% faster than `for`, just that my original statement of `for` being significantly faster is incorrect.

Reply:
I think we can safely conclude that we shouldn't be avoiding for-of because of perf concerns. Can you please update this PR? thanks!